api/RTCRtpSender.json

```diff
@@ -659,6 +659,351 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "active": {
+            "__compat": {
+              "description": "<code>parameters.encodings.active</code> parameter",
+              "support": {
+                "chrome": {
+                  "version_added": "69"
```

Comment:
Do we need to add a note for the legacy AV1 format thing mentioned here @queengooborg ? https://bugzilla.mozilla.org/show_bug.cgi?id=1230184#c31

Reply:
Looks like this was something new in Chrome 113? If you feel strongly about it, feel free to open an issue or PR. I think this PR should get in. The focus here is on dictionary removal.